tcp/mux.go

```diff
@@ -90,7 +91,9 @@ func (mux *Mux) Serve(ln net.Listener) error {
 				wg.Add(1)
 				go func(ln *listener) {
 					defer wg.Done()
-					ln.Close()
+					if err := ln.Close(); err != nil {
+						mux.Logger.Printf("tcp.Mux: Closing the listener at %s failed - %s", ln.Addr().String(), err)
```

Comment:
Nitpick: is the `.String()` suffix needed here and below? It's missing from above.